src/core/Tooltip/Tooltip.tsx

```diff
@@ -33,17 +33,8 @@ export type TooltipProps = {
  * <Button ref={buttonRef} />
  * <Tooltip content='tooltip text' reference={buttonRef} />
  */
-export const Tooltip = (props: TooltipProps) => {
-  const {
-    content,
-    children,
-    className,
-    style,
-    visible,
-    ref,
-    id,
-    ...rest
-  } = props;
+export const Tooltip = React.forwardRef<Element, TooltipProps>((props, ref) => {
```

Comment:
We always used `HTMLElement` as type. Any reason you chose Element here?

Reply:
I kept the same type as the one from tippy props

Reply:
Why I ask, cause usually ppl create refs of `HTMLElement` and then access certain props, which `Element` might not have.. And I wildly guess, that if you create `HTMLElement` ref it wont allow you to set it on Tooltip then..

Reply:
But it won't let me use HTMLElement
![image](https://user-images.githubusercontent.com/9084735/131506644-07766d13-8714-45a6-b1d4-080d811a5c77.png)